frontend/templates/troubleshooting/hooks/useTroubleshootingProps.tsx

```diff
@@ -89,6 +89,7 @@ export type TroubleshootingApiData = {
    metaDescription: string;
    metaKeywords: string;
    linkedProblems: Problem[];
+   relatedProblems: Problem[];
```

Comment:
When/where did this get added to the API?

Reply:
https://github.com/iFixit/ifixit/pull/50620

It's not gone out yet, apparently. Nothing's using the data, though.